### ISSUE TITLE
Items missing

### ISSUE THREAD

**Issue by Elementpt** [jaquadro/StorageDrawers#284]
I had this issue several times. When I craft something from the Inventory panel, some items disappear from the drawers. Ex: I want to craft a bucket and I have 64 Iron ingot. I craft 2 buckets and immediately put them inside the inventory panel, both iron and buckets vanish. Also if i set up a quarry and somehow 1 drawer gets 512 items inside it will not create another drawer with the same item to fill it until 512, and so on. That's very common with cobblestone and other items such as dirt and gravel for instance. What can i do to fix this?

**Comment by jaquadro** [comment]
Inventory panel?  Are you talking about the one in EnderIO?

**Comment by Elementpt** [comment]
Yes, that one. I tried to attach a quarry system to the drawer controll but somehow, when 1 drawer goes up to 512 (without the upgrade) it stops filling it. And don't move to another one.

Obter o Outlook para Android

On Tue, May 10, 2016 at 4:11 PM -0700, "Justin Aquadro" notifications@github.com wrote:

Inventory panel?  Are you talking about the one in EnderIO?

---
